ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/Decision/WhoDecided.cshtml.cs

```diff
@@ -0,0 +1,46 @@
+using ApplyToBecome.Data.Services;
+using ApplyToBecomeInternal.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ApplyToBecome.Data.Models.AdvisoryBoardDecision;
+
+namespace ApplyToBecomeInternal.Pages.TaskList.Decision
+{
+	public class WhoDecidedModel : DecisionBaseModel
+	{		
+		public WhoDecidedModel(IAcademyConversionProjectRepository repository, ISession session) : base(repository, session)
+		{			
+		}
+
+		[BindProperty]
+		public DecisionMadeBy DecisionMadeBy { get; set; }
+
+		public IEnumerable<DecisionMadeBy> DecisionMadeByOptions => Enum.GetValues(DecisionMadeBy.GetType())
+																	.Cast<DecisionMadeBy>();
```

Comment:
Indentation needs to be checked.

Reply:
It looks like this in VS.
![image](https://user-images.githubusercontent.com/109524605/181751253-a6fcaadd-b77e-4c2e-9c0b-dbf50d0bb6fe.png)